contracts/UFragmentsPolicy.sol

```diff
@@ -81,14 +86,24 @@ contract UFragmentsPolicy is Ownable {
         lastRebaseTimestampSec = now;
         epoch = epoch.add(1);
 
+        bool cpiValid;
+        uint256 cpi;
+        (cpi, cpiValid) = cpiOracle.getData();
+        require(cpiValid);
+
+        targetRate = cpi.mul(10 ** RATE_DECIMALS).div(TARGET_CPI);
+
+        bool rateValid;
         uint256 exchangeRate;
-        uint256 volume;
-        (exchangeRate, volume) = marketOracle.getPriceAnd24HourVolume();
+        (exchangeRate, rateValid) = marketOracle.getData();
+        require(rateValid);
```

Comment:
Note: This will change the current behavior when no fresh data exists. Current behavior rebase transaction doesn't fail and supplyDelta is set to 0. With this change rebase transaction will fail.

Reply:
Let's discuss this

Reply:
Based on our offline discussion, we are moving to a scheme where the `tx` fails with invalid data. 

We don't want to be locked out from making supply changes for upto `minRebaseTimeIntervalSec` in the case `rebase` is invoked with no fresh data from either oracle.